cython/PKGBUILD

```diff
@@ -4,13 +4,13 @@ _realname=cython
 pkgbase=${_realname}
 pkgname=('cython')
 pkgver=0.29.21
-pkgrel=1
+pkgrel=2
 pkgdesc='C-Extensions for Python'
 arch=('i686' 'x86_64')
 url='https://cython.org/'
 license=('APACHE')
 depends=('python-setuptools')
```

Comment:
An argument could be made that cython depends rather than makedepends on python-devel, as its purpose is to generate C/C++ extension code.

Reply:
On the other hand it just generates the C code and doesn't build it, right?